<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>842</width>
    <height>651</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normalon>../../40daeb43b2e3e20941ee95abdf84a311.png</normalon>
   </iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#Dialog {background-color: qlineargradient(spread:pad, x1:0.044, y1:1, x2:1, y2:0, stop:0 rgba(3, 21, 32, 255), stop:1 rgba(255, 255, 255, 255));}




</string>
  </property>
  <widget class="QCheckBox" name="filterBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTipDuration">
    <number>-8</number>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;Niagara Solid&quot;;
color: rgb(0, 255, 255);</string>
   </property>
   <property name="text">
    <string>FILTER</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>490</y>
     <width>721</width>
     <height>158</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2" rowstretch="0,0,0" columnstretch="0,0,0,0,0,0">
    <item row="2" column="1">
     <widget class="QLabel" name="secretLabel">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="addNewFilter">
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(spread:reflect, x1:1, y1:0.966, x2:0, y2:1, stop:0.238806 rgba(255, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));</string>
      </property>
      <property name="text">
       <string>Add New Filter Family</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QTextEdit" name="deviceGroupText">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>70</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">
                background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0, stop:0.199005 rgba(194, 212, 197, 255), stop:1 rgba(255, 255, 255, 255));
              </string>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="0" column="5">
     <widget class="QLabel" name="groupIDLabel">
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(spread:pad, x1:0.512, y1:1, x2:1, y2:0, stop:0 rgba(27, 93, 62, 255), stop:1 rgba(255, 255, 255, 255));
                font: 75 10pt &quot;MS Shell Dlg 2&quot;;

color: rgb(0, 255, 255);              </string>
      </property>
      <property name="text">
       <string>GROUP ID</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="deviceGroupLabel">
      <property name="styleSheet">
       <string notr="true">
background-color: qlineargradient(spread:pad, x1:0.512, y1:1, x2:1, y2:0, stop:0 rgba(27, 93, 62, 255), stop:1 rgba(255, 255, 255, 255));
                font: 75 10pt &quot;MS Shell Dlg 2&quot;;

color: rgb(0, 255, 255);              </string>
      </property>
      <property name="text">
       <string>DEVICE GROUP</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QTextEdit" name="memberText">
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>70</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0, stop:0.199005 rgba(194, 212, 197, 255), stop:1 rgba(255, 255, 255, 255));
              </string>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="1" column="5">
     <widget class="QTextEdit" name="idText">
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>70</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">
                background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0, stop:0.199005 rgba(194, 212, 197, 255), stop:1 rgba(255, 255, 255, 255));
              </string>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="memberNameLabel">
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(spread:pad, x1:0.512, y1:1, x2:1, y2:0, stop:0 rgba(27, 93, 62, 255), stop:1 rgba(255, 255, 255, 255));
                font: 75 10pt &quot;MS Shell Dlg 2&quot;;

color: rgb(0, 255, 255);              </string>
      </property>
      <property name="text">
       <string>MEMBER NAME</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>721</width>
     <height>391</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="4" column="1">
     <widget class="QLineEdit" name="deleteImg">
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(spread:pad, x1:0.512, y1:1, x2:1, y2:0, stop:0 rgba(27, 93, 62, 255), stop:1 rgba(255, 255, 255, 255));
                font: 75 10pt &quot;MS Shell Dlg 2&quot;;

color: rgb(0, 255, 255);              </string>
      </property>
      <property name="text">
       <string>DELETE</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLineEdit" name="addImg">
      <property name="styleSheet">
       <string notr="true">
          background-color: qlineargradient(spread:pad, x1:0.512, y1:1, x2:1, y2:0, stop:0 rgba(27, 93, 62, 255), stop:1 rgba(255, 255, 255, 255));
                font: 75 10pt &quot;MS Shell Dlg 2&quot;;

              
color: rgb(0, 255, 255);</string>
      </property>
      <property name="text">
       <string>ADD</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QLineEdit" name="middleLine">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 127);
color: rgb(255, 255, 255);
font: 75 8pt &quot;Microsoft New Tai Lue&quot;;</string>
      </property>
      <property name="text">
       <string>ENTER A GROUP NAME</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="groupListLabel">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>130</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(211, 211, 211);
                font: 14pt &quot;MS Shell Dlg 2&quot;;
              </string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="selectedGroupLabel">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>130</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(211, 211, 211);

              </string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QTextEdit" name="deleteFilterText">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>50</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">
                background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0, stop:0.199005 rgba(194, 212, 197, 255), stop:1 rgba(255, 255, 255, 255));
              </string>
      </property>
     </widget>
    </item>
    <item row="10" column="1">
     <widget class="QPushButton" name="deleteFilter">
      <property name="minimumSize">
       <size>
        <width>356</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>240</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(spread:reflect, x1:1, y1:0.966, x2:0, y2:1, stop:0.238806 rgba(255, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));</string>
      </property>
      <property name="text">
       <string>DELETE FILTER</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="selectedFilterImg">
      <property name="minimumSize">
       <size>
        <width>350</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>40000</width>
        <height>40000</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(spread:pad, x1:0.512, y1:1, x2:1, y2:0, stop:0 rgba(27, 93, 62, 255), stop:1 rgba(255, 255, 255, 255));
                font: 75 10pt &quot;MS Shell Dlg 2&quot;;

color: rgb(0, 255, 255);              </string>
      </property>
      <property name="text">
       <string>SELECTED FILTERS</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QTextEdit" name="addFilterText">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>50</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">
                background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0, stop:0.199005 rgba(194, 212, 197, 255), stop:1 rgba(255, 255, 255, 255));
              </string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="filterListImg">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(spread:pad, x1:0.512, y1:1, x2:1, y2:0, stop:0 rgba(27, 93, 62, 255), stop:1 rgba(255, 255, 255, 255));
                font: 75 10pt &quot;MS Shell Dlg 2&quot;;


color: rgb(0, 255, 255);              </string>
      </property>
      <property name="text">
       <string>FILTER LIST</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QPushButton" name="addFilter">
      <property name="minimumSize">
       <size>
        <width>356</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>240</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(spread:reflect, x1:1, y1:0.966, x2:0, y2:1, stop:0.238806 rgba(255, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));</string>
      </property>
      <property name="text">
       <string>ADD FILTER</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="openAddFilterScreen">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>460</y>
     <width>240</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>240</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0.044, y1:1, x2:1, y2:0, stop:0 rgba(0, 0, 127, 255), stop:1 rgba(255, 255, 255, 255));</string>
   </property>
   <property name="text">
    <string>ADDING NEW FILTER</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="ServerApplication.qrc"/>
 </resources>
 <connections/>
</ui>
